apps/src/applab/designElements/elementUtils.js

```diff
@@ -120,4 +120,13 @@ module.exports.isIdAvailable = function(newId, originalId) {
   }
 
   return true;
+};
+
+module.exports.isContentEditable = function (element) {
+  return element && element.tagName.toUpperCase() === 'DIV' && element.contentEditable;
```

Comment:
That sounds reasonable. My changes have unfortunately already been merged. Would you prefer I fix things up and let you know when I've done so, or just do this yourself as part of this PR?